CHANGELOG.md

```diff
@@ -7,6 +7,29 @@
 [//]: # (Features)
 [//]: # (BREAKING CHANGES)
 
+## December 6th, 2019
+
+### Deployment Manifest
+
+Included the generation of a Deployment Manifest artifact that can be used to promote the same application versions throughout the pipeline execution.
+
+### Air Gap Support
+
+Added new functions to support Air Gap deployment scenarios: 
+* deploy_apps_to_target_env_with_airgap - Deploy OutSystems Applications to a target environment without using the Deployment API
+* tag_apps_based_on_manifest_data - Syncronize LifeTime application versions from a Manifest file to target LifeTime
+
+### New Jenkins Templates
+
+New jenkins templates that include:
+* Lockable resources - Each stage has a semaphore to guarantee that the target environment does not have any ongoing deployment
+* Milestones - Previous builds will be discarded when reached a given pipeline milestone
```

Comment:
Ongoing builds of the same pipeline will be discarded whenever a newer build reaches a given pipeline milestone

Reply:
Done